src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml

```diff
@@ -16,10 +16,10 @@ maxSupply: '1600000000000000'
 decimals: 6
 
 treasury:
-  - addr1qxsnhcavemjjes9u6pugq8x76y3fn59t7q590qf60c4k9j5ff8jck2uprunuc8jga6ehyywd6gahfhq8f7p03exzd2aqfh7dhq
-  - addr1qymgg5j2mczqcyzq5yzv0wdzrtqv5eh8gj88r57uj0880k5rawe7srxf4duew8uxt9qjwg2hknkwc23g2gfncmqkqatsyrtzfs
-  - addr1q8e2p26qtrz02eygsgm65d04afzfwqgn0xccq3ah6haxj4tl28ezl6703wu7fhn9hax7yyp6466hn6pnamrwjenhy4cs3hmr50
-  - addr1q8rqmar56e9t05vueh5n69gdnnnfguv9m5jtvjltc6p4zu9x6n7x0yn54ckn8qwlk75pm8ewhck8kkfd56devu2lygastz5pgq
-  - addr1q93zlwt4cxge5p8fhdczt8qwrva0vuy3h88yu3lzvs55j7lsw8u030madzgeflqhsevrzdp4nls3gd0g47nef8ffj02sa7gr0t
-  - addr1qyy86us6kfrg3rsy0nhssrlkjyq4yw8wasn0n8qgk9gqp0lz3kqj9ut6p5c4gxlg7vehhkzsyfejj0gkpyf7w6d20ffqt2et79
+  - addr1qypez7msuwynwjm77vtrmmn9pm5v22y5th26exxe0kepst24k83f37z3s7fapxjamxkeedc085vg0y34zgn8x4c5m05q5nzhsy
+  - addr1q9v5l0dmy6jaz9dwkww9ghtg3yzfr6js2pejwyp6m7tjtk87hpufs2s7rtta3x4tfwa79qfht3ccxhwd7p835jytpa4qf2y00d
+  - addr1q90e9f6p69ad95998f5l9huuf70s3pd3m7yg9xynuypln3sr96ajdrxwdjdxrkayjn48qm2h35jxjdnsqlusfg027kyq97xapw
+  - addr1q9nczapcdhg7yc23ysun3tjtkx2lydkjw76pem86dw7htu69sqkqf5r066h5ca2g5qldne4yn8r3hxzm73ez6urgss3sruu6l2
+  - addr1q8swgmuz39jpknze9q8ljmwq6ndse8wpp36d8xfcxf7vkj5qp8z9c4tw4tkxer2dlsejueexww9jdp867de9xqvw6ussslysgu
+  - addr1q8v6xff20j88n7jekxw53x0g3pp7c2gcsx738q456hue6m0u9w6rhn6wgwqpsg24jfnk79lexhuzpmaj4rf9ghtw73jqmp0e04
```

Comment:
@cardanocasinovip your PR is good, but better if you can comment to explain what wallet is used for

Reply:
@0xj4m35 this wallets is our updated treasury wallets for CHIP token

Reply:
@cardanocasinovip I mean you can comment like this: 
![image](https://github.com/user-attachments/assets/fc712d83-8db7-4121-b979-d5077e212028)

Reply:
`addr1qypez7msuwynwjm77vtrmmn9pm5v22y5th26exxe0kepst24k83f37z3s7fapxjamxkeedc085vg0y34zgn8x4c5m05q5nzhsy # reserved treasury 

addr1q9v5l0dmy6jaz9dwkww9ghtg3yzfr6js2pejwyp6m7tjtk87hpufs2s7rtta3x4tfwa79qfht3ccxhwd7p835jytpa4qf2y00d # staking rewards

addr1q90e9f6p69ad95998f5l9huuf70s3pd3m7yg9xynuypln3sr96ajdrxwdjdxrkayjn48qm2h35jxjdnsqlusfg027kyq97xapw # yield farming treasury

addr1q9nczapcdhg7yc23ysun3tjtkx2lydkjw76pem86dw7htu69sqkqf5r066h5ca2g5qldne4yn8r3hxzm73ez6urgss3sruu6l2 # platform incentives

addr1q8swgmuz39jpknze9q8ljmwq6ndse8wpp36d8xfcxf7vkj5qp8z9c4tw4tkxer2dlsejueexww9jdp867de9xqvw6ussslysgu # marketing

addr1q8v6xff20j88n7jekxw53x0g3pp7c2gcsx738q456hue6m0u9w6rhn6wgwqpsg24jfnk79lexhuzpmaj4rf9ghtw73jqmp0e04 # airdrop`

Reply:
Is this better @0xj4m35 ?

Reply:
@cardanocasinovip yeah, but please apply it into your PR

Reply:
@0xj4m35 updated